stock_picking_invoice_link/views/stock_view.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- stock.picking -->
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock_picking_invoice_link.stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                    <button name="action_view_invoice" type="object"
+                        string="View Invoice" class="oe_highlight"
+                        attrs="{'invisible': [('invoice_ids', '=', [])]}"
+                        groups="base.group_user"/>
+            </field>
+            <field name="picking_type_id" position="after">
+                <field name="invoice_ids" string="Invoice"
+                    groups="account.group_account_invoice"
+                    attrs="{'invisible':[('invoice_ids', '=', False)]}"
+                    widget="many2many_tags"
+                    context="{'tree_view_ref': 'account.invoice_tree', 'form_view_ref': 'account.invoice_form'}"/>
+            </field>
+        </field>
+    </record>
+
+    <!-- stock.move -->
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">stock_picking_invoice_link.stock.move.form</field>
+        <field name="model">stock.move</field>
+        <field name="inherit_id" ref="stock.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="invoice_line_ids" widget="many2many_tags" groups="account.group_account_invoice"/>
```

Comment:
Same here, why not a tree view with more info

Reply:
I think default information is enough:

![seleccio_330](https://user-images.githubusercontent.com/4386443/30322506-ff503ad6-97b9-11e7-8d45-cf8b87628dd0.png)

Reply:
But that's the invoice form view not the stock move form view

Reply:
ok. You are right but this views is for stock_move model.

I see information is not shown so I have to correct this.

Reply:
Information in these views are shown as you can see:

![seleccio_331](https://user-images.githubusercontent.com/4386443/30323386-51ab29be-97bd-11e7-9db3-5eafce16316c.png)

![seleccio_332](https://user-images.githubusercontent.com/4386443/30323392-57694a48-97bd-11e7-9850-10966877fb96.png)

Do you think is necessary add a tree view for only show invoice line information?

Reply:
Mmm, well, I'd expect at least a reference to wich invoice this line comes from and the ability to get to it.

Reply:
@cubells here a tree view is needed for showing the minimal data:
- Invoice
- Product
- Quantity
- Price
- Taxes
- ...